static/app/views/onboarding/types.tsx

```diff
@@ -10,9 +10,9 @@ export type StepProps = {
   onComplete: (data: StepData) => void;
   onUpdate: (data: StepData) => void;
   orgId: string;
+  organization: Organization;
   platform: PlatformKey | null;
   project: Project | null;
-  organization?: Organization;
```

Comment:
I'm not sure why `organization` was marked as optional as it makes typing more difficult and the parent component has organization as required. So I just made it required.